### ISSUE TITLE
non-sanitized input variable when using filter_input_array

### ISSUE THREAD

**Issue by kkmuffme** [WordPress-Coding-Standards/WordPress-Coding-Standards#1557]
When using:
$_POST  = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
(or $_GET   = filter_input_array(INPUT_GET, FILTER_SANITIZE_STRING); with GET requests)

I still get "Detected usage of a non-sanitized input variable: $_POST". Is there a way to fix it?

The above function is especially useful, if arrays are passed in the request, since the "suggestions" for sanitizing an array are much more work intensive than using this function.

**Comment by kkmuffme** [comment]
1) yes, this is just to clarify that I mean $_POST in the example
2) why not? sorry but that's unclear to me

**Comment by jrfnl** [comment]
See: http://php.net/manual/en/filter.filters.sanitize.php

**Comment by kkmuffme** [comment]
yes, it does the same for array as for filter_input, so why should that be unsafe for array?
Yes it doesn't do the same as wordpress sanitize functions, but filter_input is fine too, isn't it?

**Comment by GaryJones** [comment]
@kkmuffme Can you please provide a full snippet of contextual code, and the full output of the report for the affected lines?
